js/meshopt_clusterizer.js

```diff
@@ -0,0 +1,443 @@
+
+	const wasmpack = new Uint8Array([
+		32, 0, 65, 2, 1, 106, 34, 33, 3, 128, 11, 4, 13, 64, 6, 253, 10, 7, 15, 116, 127, 5, 8, 12, 40, 16, 19, 54, 20, 9, 27, 255, 113, 17, 42, 67,
+		24, 23, 146, 148, 18, 14, 22, 45, 70, 69, 56, 114, 101, 21, 25, 63, 75, 136, 108, 28, 118, 29, 73, 115,
+	]);
+
+	if (typeof WebAssembly !== 'object') {
+		return {
+			supported: false,
+		};
+	}
+
+	var instance;
+
+	// for whatever reason calls to meshopt_computeClusterBounds invalidate the wasm function table for subsequent calls to meshopt_buildMeshlets(Scan) or meshopt_optimizeMeshlet
```

Comment:
I suspect there's not enough stack space and something goes awry; the current reservation is 24KB for all modules (see Makefile), and meshopt_computeMeshletBounds seems like it needs ~30 KB (meshopt_computeClusterBounds would need ~24 KB). Would probably make sense to bump this from 24 KB to 36 KB give or take...

Reply:
Ah, got it. I'll try that later this evening.

Reply:
yes, that was it. bumped it to 36kb